neqo-transport/src/connection/mod.rs

```diff
@@ -2788,6 +2841,84 @@ impl Connection {
     pub fn stream_keep_alive(&mut self, stream_id: u64, keep: bool) -> Res<()> {
         self.streams.keep_alive(stream_id.into(), keep)
     }
+
+    fn max_dgram_size_configured(&self) -> Res<u64> {
+        let tph = self.tps.borrow();
+        if let Some(r) = &tph.remote {
+            Ok(r.get_integer(tparams::MAX_DATAGRAM_FRAME_SIZE))
+        } else if let Some(r) = &tph.remote_0rtt {
+            Ok(r.get_integer(tparams::MAX_DATAGRAM_FRAME_SIZE))
+        } else {
+            Err(Error::NotAvailable)
+        }
+    }
+
+    #[cfg(test)]
+    pub fn set_max_dgram_size(&mut self, v: u64) {
```

Comment:
Do you really need this?  Or can you just set the local transport parameter?  (Note to self: check the tests that use this.)